tests/functional/p/postponed_evaluation_activated_with_alias.py

```diff
@@ -0,0 +1,28 @@
+# pylint: disable=missing-docstring,no-self-use,unused-argument,pointless-statement
+# pylint: disable=too-few-public-methods,no-name-in-module
+from __future__ import annotations as __annotations__
+
+
+class Class:
```

Comment:
I find that `Class` as the name of class is a bit confusing. Maybe a different name for the class could be clearer.